openjdk/openjdk.mk

```diff
@@ -170,14 +170,16 @@ TEST_VARIATION_JIT_PREVIEW:=
 TEST_VARIATION_JIT_AGGRESIVE:=
 TIMEOUT_HANDLER:=
 
-# if JDK_IMPL is openj9 or ibm
+# if JDK_IMPL is openj9 or ibm 
+# Prepend -Xverbosegclog for all operating systems when JDK_IMPL is openj9 or ibm
```

Comment:
I do not think we need this message. If we want to add it, it should be next to the change.

Reply:
I can remove the comment message.